apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -858,15 +880,19 @@ const Main = ({
   }
 
   const isFilteredByOnlyOneItem =
-    (filters?.teamIds?.length === 1 || filters?.userIds?.length === 1) && data.eventTypeGroups.length === 1;
+    (filters?.teamIds?.length === 1 || filters?.userIds?.length === 1) &&
+    rawData.eventTypeGroups.length === 1;
+
+  const data = denormalizePayload(rawData);
```

Comment:
Because the sent payload is in a different format now, we need to denormalize it before others can keep consuming it without any change.